native~/Runtime/src/Cesium3DTilesetImpl.cpp

```diff
@@ -426,6 +426,22 @@ void Cesium3DTilesetImpl::LoadTileset(
   TilesetContentOptions contentOptions{};
   contentOptions.generateMissingNormalsSmooth = tileset.generateSmoothNormals();
 
+  CesiumGltf::SupportedGpuCompressedPixelFormats supportedFormats;
+  supportedFormats.ETC1_RGB = tileset.ETC1_RGB();
```

Comment:
Just call `SystemInfo.IsFormatSupported(GraphicsFormat.RGB_ETC_UNorm, FormatUsage.Sample)` right here, rather than doing it in C# and then storing the value in Cesium3DTileset. You'll need to add this method to ConfigureReinterop, but I don't think that should present any issues.

Reply:
@kring  can I pass enumeration in C# code to C++ code via Reinterop ? GraphicsFormat and FormatUsage is enumeration. 
https://docs.unity3d.com/ScriptReference/Experimental.Rendering.GraphicsFormat.html
https://docs.unity3d.com/ScriptReference/Experimental.Rendering.FormatUsage.html

Reply:
Yes. Just add a call to `IsFormatSupported` to `ConfigureReinterop.cs` and you'll end up making the enums accessible to C++ as well, without even trying, because they're parameters to that function.

Reply:
Just to be clear, the line above will look a little different in C++. Something like:

```cpp
supportedFormats.ETC1_RGB = SystemInfo::IsFormatSupported(GraphicsFormat::RGB_ETC_UNorm, FormatUsage::Sample);
```

Reply:
@kring I pushed a change commit to this PR. Thanks you for your opinion. review please.